src/utils/jwt.js

```diff
@@ -8,8 +16,24 @@ const creatJWT = ({ payload }) => {
   return token;
 };
 
+/**
+ * Checks the validity of a JWT token.
+ *
+ * @param {Object} options - Options for verifying a JWT token.
+ * @param {string} options.token - The JWT token to verify.
+ * @throws {Error} - Thrown if the token is invalid.
+ */
```

Comment:
This function also returns the token object if the JWT verification passes. We can reused the type definition declared in the `checkTokenUser` file:

```suggestion
/**
 * @typedef {import('./createTokenUser.js').TokenUser} TokenUser
 */

/**
 * Checks the validity of a JWT token.
 *
 * @param {Object} options - Options for verifying a JWT token.
 * @param {string} options.token - The JWT token to verify.
 * @throws {Error} - Thrown if the token is invalid.
 * @returns {TokenUser} - User object with data for token creation.
 */
```

Reply:
We have unit tests. Is it a good practice to create documentation for them using JSDoc and integrate it into a separate HTML document for the team? For example, we currently have documentation in manual HTML format for socket tests and userById. Does it make sense to rewrite all of this using JSDoc?

Reply:
<img width="707" alt="jsDoc" src="https://github.com/Code-the-Dream-School/dd-prac-team4-back/assets/95896159/df23df10-1d9f-4b0c-af2f-60597f788bba">